tests/test_blueprint.py

```diff
@@ -95,13 +96,36 @@ def test_messages_channel(bp, app, mockredis):
             "data": '{"data": "whee", "id": "abc"}',
         }
     ]
+    pubsub.unsubscribe.side_effect = redis.exceptions.ConnectionError()
```

Comment:
You still have this `ConnectionError` in the tests, but it looks like it got dropped from the code being tested. Also, it looks like GitHub had a temporary outage, and the tests didn't get run. Can you push another commit to restore the `ConnectionError` handling, and make sure the tests run and pass?

Reply:
Good catch! I added back this code that got accidentally removed.